src/Message/p2p/AbstractRequest.php

```diff
@@ -61,4 +61,14 @@ public function setMethod($value)
     {
         return $this->setPaymentMethod($value);
     }
+
+    public function setNeed(?string $value)
+    {
+        return $this->setParameter('need', $value);
+    }
+
+    public function getNeed(): ?string
```

Comment:
```suggestion
    public function getNeed(): string
```

There is no way method will return null